api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java

```diff
@@ -5116,4 +5116,409 @@ public CohortDefinition getMQ10NUMDEN103(String flag) {
     }
     return cd;
   }
+
```

Comment:
@cfaife can you copy and past the description on top of all methods that implements query, is much easier to review, check other methods

Reply:
done